Description: Don't use NEON on arm and armhf architectures
 NEON is not required in those Debian port and usin it breaks build on
 official build servers.
 .
Author: Balint Reczey <balint@balintreczey.hu>
Forwarded: not-needed

--- a/configure.ac
+++ b/configure.ac
@@ -616,7 +616,7 @@
      CORE_SYSTEM_NAME=linux
      ARCH="arm"
      use_arch="arm"
-     use_neon=yes
+     use_neon=no
      use_gles=yes
      use_gl=no
      USE_STATIC_FFMPEG=1
